hazelcast/src/main/java/com/hazelcast/client/ClientEndpoint.java

```diff
@@ -105,6 +107,17 @@ void authenticated(ClientPrincipal principal, Credentials credentials, boolean f
      *
      * @param version The version string as obtained from the environment
      */
-    void setClientVersion(String version);
+    void setClientVersion(java.lang.String version);
```

Comment:
Are you expecting to import any other String? :) I would get rid of the java.lang